Sources/Remote Logging/Crash Logging/SentrySDK+CurrentHub.swift

```diff
@@ -1,11 +1,5 @@
 import Sentry
 
-@objc
-protocol SentrySDKInternalMethods {
-    @objc
-    var currentHub: SentryHub { get }
-}
-
```

Comment:
I moved this at the bottom of the file to make the documentation more prominent.